src/encoded/schemas/note_technical_review.json

```
@@ -0,0 +1,122 @@
+{
+    "title": "NoteTechnicalReview",
+    "description": "Schema for technical review of variants.",
+    "id": "/profiles/note_technical_review.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "identifyingProperties": [
+        "uuid"
+    ],
+    "additionalProperties": false,
+    "mixinProperties": [
+        {
+            "$ref": "mixins.json#/schema_version"
+        },
+        {
+            "$ref": "mixins.json#/uuid"
+        },
+        {
+            "$ref": "mixins.json#/attribution"
+        },
+        {
+            "$ref": "mixins.json#/submitted"
+        },
+        {
+            "$ref": "mixins.json#/modified"
+        },
+        {
+            "$ref": "mixins.json#/status"
+        },
+        {
+            "$ref": "mixins.json#/tags"
+        },
+        {
+            "$ref": "note.json#/properties"
+        }
+    ],
+    "properties": {
```

Need a schema version field here
```suggestion
    "properties": {
           "schema_version": {
                 "default": "1"
            },
```

---

No, the mixin doesn't create a default version, and it's required to add the default in to each schema or else issues with upgrading can occur. Will and I made some recent snovault fixes to patch the upgrader bug, so it shouldn't occur in the future, but still best practice is to add the default version to each item schema. There's also a test now that would have failed and caught this to try and enforce the practice. We thought about making the mixin have a default version, but there were some issues with doing so.